src/backend/distributed/sql/udfs/citus_isolation_test_session_is_blocked/latest.sql

```diff
@@ -23,10 +23,20 @@ RETURNS boolean AS $$
         FROM get_all_active_transactions() WHERE process_id = pBlockedPid;
     END IF;
 
-    RETURN EXISTS (
-      SELECT 1 FROM citus_internal_global_blocked_processes()
-        WHERE waiting_global_pid = mBlockedGlobalPid
-    );
+    IF current_setting('citus.isolation_test_check_all_blocks') = 'off' THEN
```

Comment:
I think the `if/else` deserves a comment? Something like:

In theory the IF = 'off' part should be enough, but there might be cases we have some issues in gpid processing that's why we ended up with `else` part.

Reply:
done